packages/css-data/src/property-parsers/box-shadow.test.ts

```diff
@@ -0,0 +1,73 @@
+import { describe, expect, test } from "@jest/globals";
+
+import { parseBoxShadow } from "./box-shadow";
+
+describe("parseBoxShadow", () => {
+  test("parse box-shadow from figma", () => {
```

Comment:
Test on wrong data should be added too. Its good that it parses good data, but what the result of the bad data

Reply:
tests are comping by @JayaKrishnaNamburu